homework2/task5.py

```diff
@@ -0,0 +1,15 @@
+# Реализуйте алгоритм перемешивания списка.
+
+from random import randint
+
+N = 20
+numbers = []
+for i in range(N):
+    numbers.append(randint(-N, N))
+print(numbers)
+for i in range(len(numbers)):
+    index = randint(0, len(numbers) - 1)
```

Comment:
Тут бы лучше подошла функция `randrange` модуля `random`
```suggestion
    index = randrange(N)
```